core/http-auth-aws-crt/pom.xml

```
@@ -44,6 +44,33 @@
             <artifactId>http-auth-spi</artifactId>
             <version>${awsjavasdk.version}</version>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>http-auth-aws</artifactId>
+            <version>${awsjavasdk.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk.crt</groupId>
+            <artifactId>aws-crt</artifactId>
+            <version>${awscrt.version}</version>
```

As far as I understand this should be optional, is that correct?
```suggestion
            <version>${awscrt.version}</version>
            <optional>true</optional>
```

---

Not entirely sure. I see the existing crt package (`auth-crt`) doesn't set it as optional, but optional may be better? I'll let someone from the java team weigh in.

---

http-auth-aws-crt itself is the optional dependency someone might need to pull in to get implementation of sigv4a (of which there's only the crt one), but for http-auth-aws-crt itself, aws-crt is a required dependency.

---

Yeah, thanks for clarifying, with the split I was not sure which one was it.